src/resolvers/ReviewResolver.ts

```diff
@@ -0,0 +1,277 @@
+
+            // Create new dropdown options
+            for (const dropDownInput of input.dropDown) {
+                const dropDown = DropDown.create({
+                    text: dropDownInput.text,
+                    value: dropDownInput.value,
+                    reviewQuestion: question
+                });
+                await dropDown.save();
+            }
+        }
+
+        return question;
+    }
+
+    @Mutation(() => Boolean)
+    @UseMiddleware(isAuth)
+    async deleteQuestion(@Arg("id") id: string) {
+        const question = await ReviewQuestion.findOne({ where: { id } });
+        if (!question) {
+            throw new Error("Question not found");
+        }
+
+        await question.remove();
+        return true;
+    }
+
+    // Review Mutations
+    @Mutation(() => Review)
+    async submitReview(@Arg("input") input: SubmitReviewInput) {
+        const campaign = await ReviewCampaign.findOne({ where: { id: input.campaignId } });
+        if (!campaign) {
+            throw new Error("Review campaign not found");
+        }
+
+        const review = new Review();
+        review.reviewCampaign = campaign;
+        review.dateReviewCompleted = new Date();
+
+        await review.save();
+
+        // Create answers
+        for (const answerInput of input.answers) {
+            const answer = new ReviewAnswer();
+            answer.review = review;
+            answer.questionId = { id: answerInput.questionId } as ReviewQuestion;
+            answer.questionType = answerInput.questionType;
+            answer.textAnswer = answerInput.textAnswer || "";
+            answer.rangeAnswer = answerInput.rangeAnswer || 0;
+            answer.dropDownAnswer = answerInput.dropDownAnswerId ? { id: answerInput.dropDownAnswerId } as DropDown : null;
+
+            await answer.save();
+        }
+
+        return review;
+    }
+
+    @Mutation(() => Review)
+    async updateReview(@Arg("input") input: UpdateReviewInput) {
+        const review = await Review.findOne({ where: { id: input.reviewId } });
+        if (!review) {
+            throw new Error("Review not found");
+        }
+
+        // Remove existing answers
+        await ReviewAnswer.delete({ review: { id: review.id } });
+
+        // Create new answers
+        for (const answerInput of input.answers) {
+            const answer = new ReviewAnswer();
+            answer.review = review;
+            answer.questionId = { id: answerInput.questionId } as ReviewQuestion;
+            answer.questionType = answerInput.questionType;
+            answer.textAnswer = answerInput.textAnswer || "";
+            answer.rangeAnswer = answerInput.rangeAnswer || 0;
+            answer.dropDownAnswer = answerInput.dropDownAnswerId ? { id: answerInput.dropDownAnswerId } as DropDown : null;
+
+            await answer.save();
+        }
+
+        return review;
+    }
+
+    // Review Queries
+    @Query(() => [Review])
```

Comment:
Before you can send review_campaign you need to verify user is logged in and authenticated. And your sending them the right review, and not sending them the wrong thing